### PR TITLE
Add test for double square brackets

### PR DESCRIPTION
**Pull request by ben-dyer**
Adding a test for double square brackets, added in #1947, as mentioned by @edemaine in #1954. Similar to the test added by @ice1000 in #1953 for double braces.

### CODE REVIEW

**Comment by ben-dyer** [comment]
@ice1000 Yes these were unintentional, sorry about that. Working on changing it back.

File: src/symbols.js [hunk 1]
```diff
@@ -625,6 +625,8 @@ defineSymbol(math, main, open, "[", "\\lbrack");
 defineSymbol(text, main, textord, "[", "\\lbrack");
 defineSymbol(math, main, close, "]", "\\rbrack");
 defineSymbol(text, main, textord, "]", "\\rbrack");
+defineSymbol(math, main, open, "\u27e6", "\\llbracket", true);
+defineSymbol(math, main, close, "\u27e7", "\\rrbracket", true);
```
[the remark this answers is not in the document]
Comment (ben-dyer) [reply src/symbols.js]: @kevinbarabash The formatting changes (and also this reversion) were all unintentional - I am aware of them and I am working on fixing them. I should have noticed that 4 files were changed (I only intended to edit one) before I made a PR, I will be more careful. If you think I should have closed the PR in the meantime or handled the situation somehow differently, please let me know as I am a complete beginner. My apologies for the inconvenience, and thank you for the feedback and patience!

[the remark this answers is not in the document]
Comment (kevinbarabash) [reply src/symbols.js]: @ben-dyer I missed your comment about about working on changing it back.  Sorry about that.

**Comment by ben-dyer** [comment]
I am struggling to fix this PR. I copied the current versions of the files, and then made my addition to katex-spec.js, but I am worried this is going to be giving me credit for changes made by other users. Maybe I should just delete my fork and make a new one?

File: src/symbols.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ben-dyer) [reply src/symbols.js]: Thanks @kevinbarabash. If you have a moment would you give me advice on whether I should fix this issue by deleting my current fork, closing this PR, adding my test to a new fork then making a new PR?

**Comment by ice1000** [comment]
> I am struggling to fix this PR. I copied the current versions of the files, and then made my addition to katex-spec.js, but I am worried this is going to be giving me credit for changes made by other users. Maybe I should just delete my fork and make a new one?

Nope. Because this repo always squash-merge your commits, you won't get blamed for those intermediate changes.

File: docs/supported.md [hunk 2]
```diff
@@ -104,10 +104,6 @@ The `{array}` environment does not yet support `\cline` or `\multicolumn`.
 |:----------------|:-------------------|
 | $\href{https://katex.org/}{\KaTeX}$ | `\href{https://katex.org/}{\KaTeX}` |
 | $\url{https://katex.org/}$ | `\url{https://katex.org/}` |
-| $\includegraphics[height=0.8em, totalheight=0.9em, width=0.9em, alt=KA logo]{https://cdn.kastatic.org/images/apple-touch-icon-57x57-precomposed.new.png}$ | `\includegraphics[height=0.8em, totalheight=0.9em, width=0.9em, alt=KA logo]{https://cdn.kastatic.org/images/apple-touch-icon-57x57-precomposed.new.png}` |
-
-`\includegraphics` supports`height`, `width`, `totalheight`, and `alt` in it's first argument. `height` is required.
-
```
Comment (ice1000) [inline docs/supported.md]: Why are these lines deleted?

File: docs/supported.md [hunk 3]
```diff
@@ -90,11 +90,11 @@ $( \big( \Big( \bigg( \Bigg($ `( \big( \Big( \bigg( \Bigg(`
...
 KaTeX also supports `darray`  and `dcases`.
 
-Acceptable line separators include: `\\`, `\cr`, `\\[distance]`, and `\cr[distance]`. *Distance* can be written with any of the [KaTeX units](#units).
+Acceptable line separators include: `\\`, `\cr`, `\\[distance]`, and `\cr[distance]`. *Distance* can be written with any of the [KaTeX units](#units).
 
-The `{array}` environment supports `|` and `:` vertical separators.
+The `{array}` environment supports `|` and `:` vertical separators.
 
-The `{array}` environment does not yet support `\cline` or `\multicolumn`.
+The `{array}` environment does not yet support `\cline` or `\multicolumn`.
```
Comment (ice1000) [inline docs/supported.md]: Why are there mysterious whitespace changes?

**Comment by edemaine** [comment]
@ben-dyer The huge number of bad whitespace changes suggests you might want to start over in some form (but you can still push to this branch to preserve the PR). You should be able to reset to the upstream master version via `git fetch upstream` and `git reset --hard upstream/master` (if `upstream` remote points to official repo), then redo the changes you actually want to do. [Back up your changed files before doing these commands!]

**Comment by ben-dyer** [comment]
> @ben-dyer The huge number of bad whitespace changes suggests you might want to start over in some form (but you can still push to this branch to preserve the PR). You should be able to reset to the upstream master version via `git fetch upstream` and `git reset --hard upstream/master` (if `upstream` remote points to official repo), then redo the changes you actually want to do. [Back up your changed files before doing these commands!]

@edemaine Thank you for the help! I followed your directions, but I did have to force push in order to update my fork.

**Comment by ben-dyer** [comment]
@kevinbarabash Thanks for the advice, it is much appreciated.

**Comment by ylemkimon** [comment]
[clabot:check]

**Comment by ylemkimon** [comment]
[clabot:check]
